README.md

```diff
@@ -209,24 +262,15 @@ cd trl/
 make dev
```

Comment:
btw, this fails:

```
ERROR: No matching distribution found for openrlbenchmark==0.2.1a5; extra == "dev"
```

maybe remove openrlbenchmark from dev? I think it's usually not need for dev

Reply:
Done in https://github.com/huggingface/trl/pull/2085/commits/462d9e39cd5ee7374295674eff22c470cd078208

More generally I think we should remove `openrlbenchmark` altogether? I don't think many/anyone is using it

Reply:
You can remove it. We'll add it back if necessary.

Reply:
Removed here: https://github.com/huggingface/trl/pull/2085/commits/e365eca4e0c1dca9e167b4c9eb842020fa702a88

(I wasn't sure if you'd prefer a separate PR - I can do that if you want!)

Reply:
Perfect